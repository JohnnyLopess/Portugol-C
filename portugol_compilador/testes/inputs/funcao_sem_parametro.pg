programa
{
    funcao vazio ola() {
        escreva("Oi!\n")
    }
    

    inicio
        ola()
}