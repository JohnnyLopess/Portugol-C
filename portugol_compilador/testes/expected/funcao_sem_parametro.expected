#include <stdio.h>
#include <string.h>

void ola() {
    printf("Oi!\n");
}

int main() {
    ola();
    return 0;
}
